frontend/.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": ["next/babel"],
+  "plugins": [
+    [
+      "styled-components",
+      { "ssr": true, "displayName": true, "preprocess": false }
+    ]
+  ]
+}
```

Comment:
Does this config file override nextjs's built in one? (Curiosity question)

Reply:
ummm... currently it overloads instead of override no need to worry.